src/services/VSCodeAzureSubscriptionProvider.ts

```diff
@@ -15,6 +16,13 @@ export function createVSCodeAzureSubscriptionProviderFactory(): () => Promise<VS
     }
 }
 
+export function createAzureDevOpsSubscriptionProviderFactory(): () => Promise<AzureDevOpsSubscriptionProvider> {
+    return async (): Promise<AzureDevOpsSubscriptionProvider> => {
+        azureDevOpsSubscriptionProvider ??= new AzureDevOpsSubscriptionProvider();
+        return azureDevOpsSubscriptionProvider;
+    }
+}
+

```

Comment:
This shouldn't be necessary since it's already exported from the auth package, shouldn't duplicate work here